chart/templates/issuer.yaml

```
@@ -10,5 +10,5 @@ spec:
     privateKeySecretRef:
       name: {{ .Release.Name }}-letsencrypt-issuer
     solvers:
-      {{ toYaml .Values.certificateIssuer.solvers | indent 6 }}
+{{ toYaml .Values.certificateIssuer.solvers | indent 6 }}
```

Let's do it like this, this keeps the template readable :smile:

```suggestion
      {{ toYaml .Values.certificateIssuer.solvers | nindent 6 }}
```

---

Hey @borchero ,

I literally just opened my code to change exactly this, brain want thinking properly when i put this through! very happy to make this change instead.